src/controllers/activegate/reconciler/capability/reconciler.go

```diff
@@ -56,7 +56,7 @@ func NewReconciler(capability capability.Capability, clt client.Client, apiReade
 func setReadinessProbePort() events.StatefulSetEvent {
 	return func(sts *appsv1.StatefulSet) {
 		if activeGateContainer, err := getActiveGateContainer(sts); err == nil {
-			activeGateContainer.ReadinessProbe.HTTPGet.Port = intstr.FromString(capability.HttpsServicePortName)
+			activeGateContainer.ReadinessProbe.HTTPGet.Port = intstr.FromInt(capability.HttpsServicePort)
```

Comment:
this is not in scope of this PR

Reply:
Correct, I pushed more than I intended to. It turned out I went too far with disabling ActiveGate `metrics-ingest` features if it's not defined (AG container ports are also needed for probes).